### PR TITLE
Venu/arc ds e2e

### PR DESCRIPTION
**Pull request by v-vgujjula**
azure cli downgraded and tested e2e.
Config.json is required for DS indirect mode.

### CODE REVIEW

File: testsuite/config.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
```
Comment (anraghun) [inline testsuite/config.json:1]: Why is this file required here? The sonobuoy config file is already present in the internal repo where the launcher container files are present.

**Review by anraghun — CHANGES_REQUESTED** [review]

File: testsuite/config.json (hunk 1 above)
Comment (v-vgujjula) [reply testsuite/config.json:1]: this config.json is required for Data services with Indirect mode validation. This Data services with Indirect mode is not part of launcher container files. So please keep this config.json

Comment (anraghun) [reply testsuite/config.json:1]: The [`config.json`](https://github.com/Azure/azure-arc-validation/blob/main/testsuite/config.json) was introduced only for overriding the results directory for sonobuoy versions > 0.55. Why does your config file contain all these settings?

**Review by anraghun — CHANGES_REQUESTED** [review]

File: testsuite/config.json (hunk 1 above)
Comment (v-vgujjula) [reply testsuite/config.json:1]: Unfortunately I have copied wrong file there . Now I have updated with correct file.
